gulp/tasks/copy.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+var gulp = require( 'gulp' );
+var changed = require( 'gulp-changed' );
+var replace = require( 'gulp-replace' );
```

Comment:
Lazy loads the plugins as they're needed and reduces the `require()`s you need to keep up in your files https://www.npmjs.com/package/gulp-load-plugins